lib/tasks/doctor/check-npm-outdated.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
```

Comment:
Are these local ember-addons, local to Ember CLI it self or in the app blueprint?  If the former is there any example of this?